ddtrace/tracer/textmap_test.go

```diff
@@ -226,20 +226,50 @@ func TestB3(t *testing.T) {
 		os.Setenv("DD_PROPAGATION_STYLE_EXTRACT", "b3")
 		defer os.Unsetenv("DD_PROPAGATION_STYLE_EXTRACT")
 
-		headers := TextMapCarrier(map[string]string{
-			b3TraceIDHeader: "1",
-			b3SpanIDHeader:  "1",
-		})
-
-		tracer := newTracer()
-		assert := assert.New(t)
-		ctx, err := tracer.Extract(headers)
-		assert.Nil(err)
-		sctx, ok := ctx.(*spanContext)
-		assert.True(ok)
+		var tests = []struct {
+			label string
+			in    TextMapCarrier
+			out   []uint64 // contains [<trace_id>, <span_id>]
+		}{
+			{
+				"only_ones",
```

Comment:
We can remove the “label” part. You can just use the empty string for the test name and Go will automatically fill it in using numbers.

Reply:
That's fine, and the labels are removed. 👍